boards/hail/README.md

```diff
@@ -40,3 +111,26 @@ You can also specify the serial port to use:
 ```bash
 make program PORT=/dev/ttyUSB0
 ```
+
+Under Mac OS X, plug in the board and use `ls -ltr /dev/tty*` to find a recently created device:
```

Comment:
I think @cxbrooks may have been shipped a hail board that had not been properly provisioned (as was I). A fully provisioned hail should have the FTDI firmware modified to have a product name which includes the string "Tock" and `tockloader` (which was written for hail) looks for such a device and chooses it by default.

So on a properly provisioned hail you shouldn't need any of this section. Is that right @bradjc @brghena?